src/main/java/frc/robot/RobotContainer.java

```diff
@@ -280,12 +291,12 @@ private void configureButtonBindings() {
                             () -> -driver.getRawAxis(leftY), 
                             () -> -driver.getRawAxis(leftX), 
                             () -> driver.getRawAxis(rightX),
-                            () -> driverDpadUp.getAsBoolean(),
+                            () -> false,
                             () -> s_Eyes.getTargetRotation(),
-                            () -> driverB.getAsBoolean(),
+                            () -> false,//driverB.getAsBoolean(),
                             rotationSpeed,
-                            true
-                        ).alongWith(new AimShoot(s_Eyes, s_ShooterPivot, s_Shooter, true)),
+                            false //TODO Determine if we want auto rotation aiming
```

Comment:
remove TODO comment and change to true. Drive team wants auto aim while using the elevator shot